src/Microsoft.HttpRepl.Tests/Commands/GetCommandTests.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Tests.SampleApi;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class GetCommandTests : IClassFixture<GetCommandsFixture>
+    {
+        private readonly GetCommandsConfig _config;
+        public GetCommandTests(GetCommandsFixture getCommandsFixture)
+        {
+            _config = getCommandsFixture.Config;
+        }
+
+        [Fact]
+        public async Task NoBasePath_ReturnsMessage()
+        {
+            HttpState httpState = new HttpState();
+
+            string expectedErrorMessage = StringResources.Error_NoBasePath.SetColor(httpState.ErrorColor);
+            string actualErrorMessage = null;
+
+            IShellState shellState = MockHelpers.GetMockedShellState(errorMessageCallback: (s) => actualErrorMessage = s);
+
+            GetCommand command = new GetCommand();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("GET");
+            CancellationTokenSource cts = new CancellationTokenSource();
+
+            await command.ExecuteAsync(shellState, httpState, parseResult, cts.Token);
+
+            Assert.Equal(expectedErrorMessage, actualErrorMessage);
+        }
+
+        [Fact]
+        public async Task MultipartRoute()
```

Comment:
Per Jimmy's suggestion in one of the code reviews, I've been trying to follow this naming scheme - <MethodBeingTested>_<DescribeInputScenario>_<DescribeExpectedResult>. It makes the test name clear and it's easier to tell what scenario is being tested without looking at the test implementation.

Reply:
Changed in e3302fb